game.py

```diff
@@ -20,7 +20,38 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    user_wins = is_word_guessed(snowman_word, correct_letter_guess_statuses)
+
+    while (
+        (len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES)
+        and (not user_wins)
+    ):
```

Comment:
Love that you are breaking up your code onto separate lines to ensure that it doesn't go pass the conventional 80 character limit for readability. However, some of the surrounding whitespace is not necessary. You can read more about whitespace/indentation and the conventions to follow while using it on the [PEP 8 website](https://peps.python.org/pep-0008/#blank-lines). (**Note**: This line of code could be on a single line since it doesn't exceed 80 characters). 
```suggestion
    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not user_wins:
```

Reply:
I accidentally internalized limiting to 72 characters!!! The recommended limit being 80 is going to make my life so much easier.